api/main.py

```diff
@@ -26,19 +31,72 @@
     CORSMiddleware,
     allow_origins=ORIGINS,
     allow_credentials=True,
-    allow_methods=["GET", "POST"],
+    allow_methods=["GET", "POST", "PUT", "DELETE"],
     allow_headers=["*"],
 )
 
 
+# Dependency
+def get_db_session():
```

Comment:
I feel like this should be in database.py? Opinion?

Reply:
@Sybrand  I referred to fastapi documentation and setup this and I kind like this idea of having session in main rather than database.